doc/known_issues.rst

```diff
@@ -59,6 +67,27 @@ KRKNWK-20863: Increased boot time due to bootloader configuration
   
   **Affected platforms:** All platforms (most significant impact on nRF52840)
 
+KRKNWK-20856 GNSS scan enters into endless loop
+
+  **Affected platforms:** nRF52840, nRF54L15 with LR1110 shield
+  
+  **Note:** The reproduction of this issue is not consistent.
+  
+  **Impact:** Small - location scans are mostly for test purposes. Issue does not occur with location send command.
+
+KRKNWK-20857 Location modem stays busy forever when scan operation overlaps with deinit
+
+  **Affected platforms:** nRF52840, nRF54L15 with LR1110 shield
+
+  **Impact:** Small - uncommon test scenario, location reinitialized after deinitialized during scan.
+
+
+KRKNWK-21159 Crash in sid_location_run due to invalid context in Wi-Fi callback
+
+  **Affected platforms:** nRF52840, nRF54L15 with LR1110 shield
+
+  **Impact:** Moderate
```

Comment:
Could we please add the workaround paragraph and paste the patch?

I think we should do the same with the issue with IRQ we approached recently, with proposed patch as a current workaround.